fixtures/workers-with-assets-and-service-bindings/tests/index.test.ts

```diff
@@ -127,8 +131,8 @@ describe.each(devCmds)(
 			describe("Service binding to default entrypoint", () => {
 				// this currently incorrectly returns the User Worker response
 				// instead of the Asset Worker response
-				it.fails(
-					"should return Asset Worker response for fetch requestsfor routes that serve static content",
+				failsIf(!args.length)(
```

Comment:
same as above 🙂

Reply:
Also (VERY NIT! 🙈) I think something like this would be much more readable/easy to follow without having to check other parts of this file: `failsIf(!args.includes("--x-assets-rpc"))`

Reply:
totally fair. We are removing the flag though this week, and once we do that, all these `failIf`s won't be necessary anymore

Reply:
ah nice ok! 😄